SETUP/tests/smoketests/pageload_smoketest.py

```diff
@@ -667,6 +704,26 @@ def main() -> int:
             ret = 1
             break
 
+    for test in CRON_JOBS:
+        print(f"Cronjob: {test}")
+        cmd_result = run(
+            [
+                "php",
+                os.path.join(config['code_dir'], "crontab", "run_background_job.php"),
+                test,
+                "true"
+            ],
+            stdout=PIPE,
+            stderr=STDOUT,
+            encoding='utf-8'
+        )
+        if cmd_result.returncode != 0 or not 'Succeeded: true' in cmd_result.stdout or test_failed(logs):
```

Comment:
I don't think `logs` is updated in this loop (it is pulled from the PHP server in the web tests) so we can remove the check here and dumping it on error (l. 723).

Reply:
Indeed. I assumed (incorrectly) this was tied to the system `php_errors` but it isn't. I also discovered that we were relying on the default error level being E_ALL which it isn't. I've added a commit that explicitly sets the error level and display errors config. This resulted in PHP 8.3 showing deprecation errors that we'll need to resolve, so for now I've disabled the PHP 8.3 smoke tests.